# **BREAST CANCER**

## **IMPORT LIBRARY**

In [6]:
import pandas as pd

## **DATASET**

In [11]:
path = "/content/drive/MyDrive/Dataset Praktikum AVD Octaa/Breast_Cancer.csv"

df = pd.read_csv(path)

df

Age   Race Marital Status T Stage  N Stage 6th Stage  \
0      68  White        Married       T1      N1       IIA   
1      50  White        Married       T2      N2      IIIA   
2      58  White       Divorced       T3      N3      IIIC   
3      58  White        Married       T1      N1       IIA   
4      47  White        Married       T2      N1       IIB   
...   ...    ...            ...      ...     ...       ...   
4019   62  Other        Married       T1      N1       IIA   
4020   56  White       Divorced       T2      N2      IIIA   
4021   68  White        Married       T2      N1       IIB   
4022   58  Black       Divorced       T2      N1       IIB   
4023   46  White        Married       T2      N1       IIB   

                  differentiate Grade   A Stage  Tumor Size Estrogen Status  \
0         Poorly differentiated     3  Regional           4        Positive   
1     Moderately differentiated     2  Regional          35        Positive   
2     Moderately differentiated     2  Regional          63        Positive   
3         Poorly differentiated     3  Regional          18        Positive   
4         Poorly differentiated     3  Regional          41        Positive   
...                         ...   ...       ...         ...             ...   
4019  Moderately differentiated     2  Regional           9        Positive   
4020  Moderately differentiated     2  Regional          46        Positive   
4021  Moderately differentiated     2  Regional          22        Positive   
4022  Moderately differentiated     2  Regional          44        Positive   
4023  Moderately differentiated     2  Regional          30        Positive   

     Progesterone Status  Regional Node Examined  Reginol Node Positive  \
0               Positive                      24                      1   
1               Positive                      14                      5   
2               Positive                      14                      7   
3               Positive                       2                      1   
4               Positive                       3                      1   
...                  ...                     ...                    ...   
4019            Positive                       1                      1   
4020            Positive                      14                      8   
4021            Negative                      11                      3   
4022            Positive                      11                      1   
4023            Positive                       7                      2   

      Survival Months Status  
0                  60  Alive  
1                  62  Alive  
2                  75  Alive  
3                  84  Alive  
4                  50  Alive  
...               ...    ...  
4019               49  Alive  
4020               69  Alive  
4021               69  Alive  
4022               72  Alive  
4023              100  Alive  

[4024 rows x 16 columns]

## **STUDY CASE**

Sebuah rumah sakit ingin meneliti faktor utama yang memengaruhi harapan hidup pasien penderita kanker payudara. Rumah sakit tersebut memiliki data pasien yang mencakup stadium kanker berdasarkan sistem *American Joint Comite on Cancer* (AJCC) edisi ke-6, ukuran tumor, jumlah kelenjar bening yang positif, status hormon (Estrogen dan Progenteron), dan lama bertahan hidup pasien dalam bulan.

Pihak rumah sakit ingin mengetahui apakah tingkat keparahan kanker, ukuran tumor, dan penyebaran ke kelenjar getah bening benar-benar berpengaruh terhadap harapan hidup pasien. Dengan menggunakan dataset yang berasal dari SEER Program milik National Cancer Institute, analisis dilakukan terhadap 4024 pasien perempuan yang didiagnosis kanker payudara pada tahun 2006–2010. Data tersebut digunakan untuk menganalisis hubungan antara stadium kanker (*T Stage, N Stage, 6th Stage*), ukuran tumor, jumlah kelenjar getah bening positif (*Regional Node Positive*), dan status hormon dengan tingkat harapan hidup pasien.

## **BUSINESS UNDERSTANDING**

### **Business Objective**

Dataset ini berisi data yang diperoleh dari pasien kanker payudara yang mencakup informasi stadium kanker yang telah disesuaikan berdasarkan sistem AJCC (*American Joint Committee on Cancer*) edisi ke-6, ukuran tumor, penyebaran ke kelenjar getah bening, status hormon, dan harapan hidup pasien.

Permasalahan utamanya adalah bagaimana karakteristik kanker yang di derita oleh pasien memengaruhi harapan hidup pasien, serta bagaimana faktor-faktor seperti **stadium kanker** (*T Stage, N Stage, 6th Stage*), **ukuran tumor**, dan **jumlah kelenjar getah bening positif** (*Regional Node Positive*) berkontribusi terhadap perkiraan kondisi pasien.

Tujuan analisis:  
*   Menganalisis faktor-faktor yang paling memengaruhi kelangsungan hidup pasien
*   Sebagai data pendukung dalam evaluasi dan pengambilan keputusan terkait tingkat risiko pasien

### **Asses Situation**

Dataset pasien kanker payudara ini diperoleh dari pembaruan November 2017 Program SEER milik NCI (*National Cancer Institute*), yang menyediakan informasi tentang statistik kanker berbasis populasi di Amerika Serikat. Dataset ini melibatkan pasien wanita dengan kanker payudara **karsinoma duktus** (*Infiltrating duct*) dan **lobular infiltratif** (*Lobular carcinoma*) yang didiagnosis pada tahun 2006-2010. Pasien dengan ukuran tumor yang tidak diketahui, kelenjar getah bening regional yang diperiksa, kelenjar getah bening regional positif, dan pasien yang masa hidupnya kurang dari 1 bulan dikecualikan; dengan demikian, 4024 pasien akhirnya dimasukkan.

### **Analytic Goals**

Berdasarkan permasalahan di atas, tujuan dari dilakukannya analisis tersebut, yaitu:
* Menganalisis data untuk memahami hubungan antara karakteristik kanker yang diderita pasien dengan tingkat harapan hidup pasien
* Melihat data secara menyeluruh untuk mengetahui pola dan hubungan antar variabel
* Mengidentifikasi variabel yang paling berpengaruh terhadap tingkat harapan hidup

### **Project Plan**

Setelah menentukan tujuan analisis, rencana tahapan proyek yang akan dilakukan:

1.   Memahami Dataset
     *   Melihat dan memahami struktur dataset
     *   Memeriksa jenis data yang terdapat di dalam dataset
     *   Memeriksa keutuhan data
2.   Mengolah Data
     *   Menghapus data yang salah atau *double*
     *   Memperbaiki data jika ada yang salah
     *   Mengubah data kategori menjadi angka
3.   Eksplorasi Data
     *   Membuat grafik berdasarkan dataset untuk melihat distribusi datanya
     *   Melihat hubungan antar setiap variable
     *   Membandingkan pasien dengan harapan hidup tinggi dan rendah
4.   Kesimpulan
     *   Menyimpulkan hasil analisis yang telah dilakukan
     *   Menjelaskan faktor yang paling memengaruhi harapan hidup penderita kanker payudara

